cmd/static/default-config.toml

```diff
@@ -101,6 +101,18 @@ TargetMap = []
 # ["IP:{{SOURCE_IP}}", "Proxy:lets-proxy", "Protocol:{{HTTP_PROTO}}" ]
 Headers = [ "X-Forwarded-Proto:{{HTTP_PROTO}}", "X-Forwarded-For:{{SOURCE_IP}}" ]
 
+# A map with an IP key/mask and a value with an array of strings separated by a colon Header:Header Value
```

Comment:
And remove/comment header for 127.0.0.1 - it will no do real work, but can enable additional director for every request.